src/main/java/se/sundsvall/casedata/integration/db/listeners/AppealListener.java

```diff
@@ -1,10 +1,11 @@
 package se.sundsvall.casedata.integration.db.listeners;
 
-import org.springframework.stereotype.Component;
-
 import jakarta.persistence.PostPersist;
 import jakarta.persistence.PreRemove;
 import jakarta.persistence.PreUpdate;
+
+import org.springframework.stereotype.Component;
+
 import se.sundsvall.casedata.integration.db.model.Appeal;
 
 @Component
```

Comment:
Annotation not necessary (comment applicable for all entity listeners in this package)

Reply:
Would be true if the listener(s) didn't have a dependency on ErrandListener.